cpp/tensorrt_llm/kernels/trtllmGenKernels/fmha/kernelParams.h

```diff
@@ -781,8 +801,7 @@ struct KernelParams
         params.mNumHiddenEltsO = options.mNumHeadsQ * options.mHeadDimQk;
         params.mOutputScale = 1.f;
         params.mScaleSoftmaxLog2 = (1.f / (std::sqrt((float) (options.mHeadDimQk)) * options.mScaleQ)) * M_LOG2E;
-        params.mStartTokenIdxSfO = options.mSfStartTokenIdx;
-        params.mScaleSfKv = options.mScaleSfKv;
```

Comment:
Wondering why this global scale is not needed any more ? or it has been fused into the SFs.

Reply:
or it is fused into qkv_scale_quant_orig.

Reply:
does it mean that the fmha kernel doesn't read from mScaleSfKv ?

Reply:
You are right. The kernel does not use mScaleSfKv for now.